reblog/basics.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
This notebook doesn't render great in a review, but I've seen the output results which look like a good result.